clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/actions/AttachTitusServiceLoadBalancers.java

```diff
@@ -62,9 +63,7 @@ public Result apply(@Nonnull AttachTitusServiceLoadBalancersCommand command, @No
         titusClientProvider.getTitusLoadBalancerClient(
             description.getCredentials(), description.getRegion());
     if (loadBalancerClient == null) {
-      // TODO(rz): This definitely doesn't seem like something to casually skip?
-      saga.log("Unable to create load balancing client in target account/region");
-      return new Result();
+      throw new TitusException("Unable to create load balancing client in target account/region");
```

Comment:
This is an unrelated change, but while I was poking around, it seemed like a bad thing to keep. I'd rather bomb out the saga and let it retry instead of happily skipping if the load balancer client can't be created (for whatever reason).

I believe this original behavior was carried over from the original, pre-saga, Groovy code.